resources/core/charts/console-backend-service/templates/deployment.yaml

```diff
@@ -80,8 +80,10 @@ spec:
             value: "https://dex.{{ .Values.global.ingress.domainName }}"
```

Comment:
Suggestion: If we are doing multiple ifs that all depend on the same condition (isGardener here), it may be better to have 2 depoyment files (deployment and deployment-gardener) and pick which one we use.